### ISSUE TITLE
A11y_SQLAzureDataStudio_Server Dashboard_Server Dashboard_ScreenReader: Screenreader doesn't announce anything for retrieved searched query result

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9208]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

GitHubTags:#A11yWCAG2.1;#A11yTCS;#SQL Azure Data Studio;#DesktopWeb;#Win32;#MAS4.1.3;#JAWS;#NVDA;#Benchmark;#A11y_SQLAzureDataStudioBenchmark;#A11ySev2;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

Additional Details:
Screen reader: NVDA(2019.2), JAWS(2019.1912.1 ILM)
MAS reference: MAS4.1.3

**Repro Steps:**
1.	Launch Azure Data Studio.
2.	Connect to server.
3.	In the Connections pane, right-click on a server & select Manage.
4.	Launch JAWS screenreader.
5.	Navigate to ‘Search’ section- Editbox using tab key
6.	Enter some keywords.

**Actual:**
While JAWS is on and navigated to ‘Search’ Editbox, and entered keywords, System will display results according to query, but the same is not announced by screenreader. Screenreader doesn’t announce any update to user.
Similar issue has been observed with NVDA also.

**Expected:**
Screenreader should announce details of fetched results of the searched query. Screen reader should announce number of records found.

**User Impact:**
Screen reader user will not be able to get the information of searched query result.

**Attachment link reference:**
[9208_A11y_SQLAzureDataStudio_Server Dashboard_Server Dashboard_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4274916/9208_A11y_SQLAzureDataStudio_Server.Dashboard_Server.Dashboard_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
search result should be announced

**Comment by fsteffi** [comment]
GitHubTags:#A11y_SQlADS_WCAG2.1;

**Comment by sspatil123** [comment]
**GithubTags**:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
